### PR TITLE
, fix :)

### PR DESCRIPTION
**Pull request by LoweredgamesDev**
missing . in Readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 > [!WARNING]
 > This project is a prototype for a shippable Bevy Editor and is still in heavy development and testing. 
-> Everything you see is still to be working or cleaned up. read about what we're building in the [Design Book](https://bevyengine.github.io/bevy_editor_prototypes/)! or [Figma document]
+> Everything you see is still to be working or cleaned up. read about what we're building in the [Design Book](https://bevyengine.github.io/bevy_editor_prototypes/)! or [Figma document],
```
Comment (alice-i-cecile) [inline README.md:5]:
```suggestion
> Everything you see is very much a work-in-progress. Read about what we're building in the [Design Book](https://bevyengine.github.io/bevy_editor_prototypes/)! or [Figma document],
```

File: README.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
...
 > to run just type this ```cargo run``` or ```cargo run --example simple_editor``` command in the terminal of the ```bevy_editor_prototypes folder```
```
Comment (alice-i-cecile) [inline README.md:6]:
```suggestion
> To run just type this ```cargo run``` or ```cargo run --example simple_editor``` command in the terminal of the ```bevy_editor_prototypes folder```
```

Comment (Loweredgames) [reply README.md]: 👍

**Comment by LoweredgamesDev** [comment]
you can merge if you want

**Pull request merged**